lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -1060,6 +1086,80 @@ def _restart_opensearch(self, event: _RestartOpenSearch) -> None:
 
         self._start_opensearch_event.emit()
 
+    def _settle_voting_exclusions(self, unit_is_stopping: bool = False):  # noqa C901
```

Comment:
Could we move this to the `opensearch_nodes_exclusions` module? and rename it as `settle_voting`?